persistence/azure-storage/index.md

```diff
@@ -15,6 +15,19 @@ redirects:
 
 Certain features of NServiceBus require persistence to permanently store data. Among them are subscription storage, sagas, and timeouts. Various storage options are available including Azure Storage Services.
 
+Azure Storage Persistence stores NServiceBus data in [Azure Table storage](https://azure.microsoft.com/en-us/services/storage/tables/).
+
+## Persistence at a glance
+
+|Feature                    |   |
+|:---                       |---
+|Supported Storage Types    |Sagas, Subscriptions, Timeouts
+|Unsupported Storage Types  |Outbox
+|Transactions               |None
```

Comment:
For all sections the following applies:

Shouldn't this be split in 3?

Locking: Pessimistic, Optimistic, None / Last writer wins
Transactional: Full, Atomic, No
Transport transaction sharing: Distributed transactions (MSDTC), Connection sharing, No

Reply:
Locking, I handle via saga concurrency control on the next line.

I expanded transactions via the legend linked above to explain that it describes how saga/outbox acts and that if it's transactional, there's a way to share that transaction via synchronized storage session.